### ISSUE TITLE
[15pt] Develop metadata reporting system

### ISSUE THREAD

**Issue by BradfordBates-NOAA** [NOAA-OWP/ras2fim#46]
Once ras2fim-derived maps are used by the field, we will receive a lot of questions about model source, accuracy, etc. Furthermore, since there are likely inherent errors or model shortcomings that we come across while using ras2fim to produce REMs, we should aim to document said errors/shortcomings.

Here are some of the possible attributes we could track and include in deliveries to hydrovis. These would preferably be included in the attributes of the data that we deliver. @JamesColl-NOAA @CarsonPruitt-NOAA @AliForghani-NOAA @LauraKeys-NOAA @RobHanna-NOAA please feel free to add more attributes as you think of them.

**At a minimum:**
- [ ] Data provider, e.g. BLE (for Beta delivery this could be hardcoded)
- [ ] ras2fim version (this one is critical) -- use Changelog version
- [ ] ras2fim processing date (when the model was run through ras2fim)
- [ ] Rating curve range (e.g. 0-35ft or 0-25,000cfs)
- [ ] Relevant `feature_id`
- [ ] Model info (e.g. 1D, 2D)

**Nice to haves:**
- [ ] Terrain match score (@JamesColl-NOAA's idea)
- [ ] Flow-matching characteristics success (@JamesColl-NOAA's idea)
- [ ] Developer note in plain English, e.g. "This model appears to have X, Y, and Z shortcomings."
- [ ] When the model was last updated (if available)

I can envision these attributes being added to a model footprint polygon layer that is delivered to Hydrovis and published alongside the FIM services. I think Jim also said this.

**Comment by LauraKeys-NOAA** [comment]
Great start on this list. Should this metadata, when possible, be recorded in the catchment polygons that get generated in the ras2rem process? Or should there be some separate metadata layer? I also imagine the rating curve range info should wait until the inundation (ras2fimundation?) step.

**Comment by BradfordBates-NOAA** [comment]
Let's discuss tomorrow but I could see the catchment geometry being a nice way to track attributes, especially catchment-specific ones like rating curve info.

**Comment by BradfordBates-NOAA** [comment]
@CoreyKrewson-NOAA Please see this ticket and provide any thoughts you have.

**Comment by CoreyKrewson-NOAA** [comment]
When you say "Relevant hydroid" do you mean "Relevant NWM feature ID"?
